### ISSUE TITLE
Neovim freezing after running tests while jest is not installed

### ISSUE THREAD

**Issue by lars-vc** [nvim-neotest/neotest-jest#56]
I ran into this problem after forgetting to run `yarn install`, so jest wasn't installed yet. But running the tests just froze neovim completely, I had to kill the terminal. It would be nicer to have an error message using `vim.notify`, since the user is unable to tell what is going on. I see this is perhaps resolved in https://github.com/haydenmeade/neotest-jest/pull/37, but this pr hasn't seen activity in a while.

**Comment by mikaoelitiana** [comment]
hello @lars-vc, do you still experience those freezing? If not, I think we should close this issue

**Comment by lars-vc** [comment]
Yes, this issue still persists. Still freezes when jest is not installed.

**Comment by rip222** [comment]
yep it's breaking. I had to turn off a lot of plugins to determine it was this crap ruining everything
